spec/integration/guest_view_spec.rb

```diff
@@ -47,7 +51,6 @@
 
     it "shows the loaded image for each item" do
       create_item
-
```

Comment:
Sorry. I meant that the black line should be left in there to show the separation setup and exercise steps.